playbooks/kubevirt-web-ui/private/config.yml

```diff
@@ -12,18 +12,19 @@
           playbook: "playbooks/kubevirt-web-ui/config.yml"
           status: "In Progress"
           start: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
+    when: cluster | default("openshift") =="openshift"
```

Comment:
you shouldn't have to worry about cluster being set.  If you're seeing not getting set, then cluster should be defaulted to openshift elsewhere.

Reply:
ok, removed from here, the variable must be set to install Kubevirt Web UI.
I've added a debug message on the top of this file if `cluster` is missing.

Reply:
CI is failing on undefined `cluster` during uninstallation.

@rthallisey , are you fine with adding this default back? (see 2nd commit)
To fix deprovision phase farther, I've cherry-picked #423 here.